lib/genesis_collector/disks.rb

```diff
@@ -29,9 +29,8 @@ def collect_disks
     private
 
     def disk_healthy(disk)
-      short_tests = shellout_with_timeout("smartctl -H #{disk}", 5)
-
-      short_tests =~ /result: PASSED/
+      shellout_with_timeout("smartctl -H #{disk}", 5)
+      $CHILD_STATUS.success?
```

Comment:
Given the current patterns, I think this is ok. I'm not a huge fan of how we're doing `shellout_with_timeout` but we can fix that later.